### ISSUE TITLE
Unuseful error when editing a username

### ISSUE THREAD

**Issue by PMQABen** [ProcessMaker/bpm#1258]
1. Open `/admin/users/`
2. Create a new user
3. Edit user, change username to put the `<=` symbols in it
4. Click save

**Result:**
1. Notice message: `An error occurred while saving the Groups.`
2. Console error 422:
```
﻿PUT http://bpm4.local.processmaker.com/api/1.0/users/3 422 (Unprocessable Entity)
vendor.js?id=03b5b59…:188
```

**Expected:**
1. A notice message that explains the error is due to a/an invalid character/characters in the username field.
2. Helper text on the username field that references what characters are invalid. _(optional)_

**Comment by nolanpro** [comment]
Will be fixed as part of #1289

**Comment by nolanpro** [comment]
Fixed in #1289
